src/executables/csql.c

```diff
@@ -1915,11 +1943,13 @@ csql_execute_statements (const CSQL_ARGUMENT * csql_arg, int type, const void *s
 		{
 		  db_abort_transaction ();
 		  do_abort_transaction = false;
+		  cub_ddl_log_msg ("auto_rollback");
```

Comment:
Since it is a hard coding style, it is recommended to modify it as following:
in the line csql.c:137 insert two lines
```
char cub_ddl_log_auto_commit_msg[PATH_MAX] = "auto_commit";
char cub_ddl_log_auto_rollback_msg[PATH_MAX] = "auto_rollback";
```
and modify it to

`cub_ddl_log_msg (cub_ddl_log_auto_rollback_msg);`

It's your choice.

Reply:
Thank you, I will change it to the #define statement.